src/main/kotlin/turfkt/Midpoint.kt

```diff
@@ -0,0 +1,28 @@
+package turfkt
+
+import geojsonkt.*
+import turfkt.*
+
+fun Position.midpoint(to: Position): Position {
+    val dist = this.distance(to)
+    val heading = this.bearing(to)
+    val units = "" // not sure what to have this default to but i noticed it was not used at all in this instance
```

Comment:
Remove `units`. It should be an optional parameter in `destination`, so you don't need to provide since it.